modules/ui/combobox.js

```diff
@@ -386,7 +386,17 @@ export function uiCombobox(context, klass) {
                     return 'combobox-option ' + (d.klass || '');
                 })
                 .attr('title', function(d) { return d.title; })
-                .html(function(d) { return d.display || d.value; })
+                .html(function(d) {
+                    // d.display can be an object
+                    if ( typeof d.display === 'object' &&
+                         !Array.isArray(d.display) &&
+                         d.display !== null
+                    ) {
+                        return d.display.label.value;
+                    }
+
+                    return d.display || d.value;
+                })
```

Comment:
The `display` object’s `language` property would be a suitable value for the element’s `lang` attribute.

Reply:
True but isn't this part of code just responsible to display dropdown values and not assign anything.
Or am I understanding it completely wrong?